bibliopixel/animation/animation.py

```diff
@@ -138,15 +138,15 @@ def run_all_frames(self):
                 self.run_one_frame()
 
     def set_runner(self, runner):
-        self.runner = runner
+        self.runner = Runner(**(runner or {}))
         self.threading = AnimationThreading(self.runner, self.run_all_frames)
 
     def start(self):
         self.threading.start()
 
     def run(self, **kwds):
         # DEPRECATED
-        self.set_runner(Runner(**kwds))
+        self.set_runner(kwds)
```

Comment:
Why should other classes need to know about the existence of Runner, which they never see?